docs/main.js

```diff
@@ -0,0 +1,49 @@
+document.getElementById('contactForm').addEventListener('submit', function(event) {
+    // Clear previous error messages
+    const errorMessageDiv = document.getElementById('error-message');
+    errorMessageDiv.textContent = '';
```

Comment:
You don't have ` <div>` with the id `error-message` in your HTML, so `errorMessageDiv` will be `null`, which causes an error when you try to set `errorMessageDiv.textContent`